src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -106,30 +84,22 @@ function dismissWorkspaceError(policyID, pendingAction) {
     throw new Error('Not implemented');
 }
 
-function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, userWallet}) {
+function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, userWallet}: WorkspaceListPageProps) {
     const theme = useTheme();
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     const {isOffline} = useNetwork();
 
     /**
-     * @param {Boolean} isPaymentItem whether the item being rendered is the payments menu item
-     * @returns {String|undefined} the user's wallet balance
+     Get the user's wallet balance
      */
-    function getWalletBalance(isPaymentItem) {
-        return isPaymentItem ? CurrencyUtils.convertToDisplayString(userWallet.currentBalance) : undefined;
-    }
+    const getWalletBalance = (isPaymentItem: boolean) => (isPaymentItem ? CurrencyUtils.convertToDisplayString(userWallet?.currentBalance) : undefined);
 
     /**
      * Gets the menu item for each workspace
-     *
-     * @param {Object} item
-     * @param {Number} index
-     * @returns {JSX}
      */
-    function getMenuItem(item, index) {
-        const keyTitle = item.translationKey ? translate(item.translationKey) : item.title;
-        const isPaymentItem = item.translationKey === 'common.wallet';
```

Comment:
Please check in runtime that translationKey isn't defined on items. I'm good to remove it but I think along with this line you should be able to remove `getWalletBalance` util, or at least refactor it.

Reply:
I tested, the ```item.translationLey``` is always ```undefined```

Reply:
Also removed the ```getWalletBalance```